src/test/java/com/qualimente/training/addressbook/AddressTest.java

```diff
@@ -67,15 +74,16 @@ public void copyWith_copies_instance_properly() {
   @Test
   public void implement_equals_for_domain() {
     String id = anyString();
+    String name = anyString();
     String line1 = anyString();
     String line2 = anyString();
     String city = anyString();
     String postalCode = anyString();
     String state = anyString();
     String country = anyString();
 
-    Address address1 = new Address(id, line1, line2, city, postalCode, state, country);
-    Address address2 = new Address(id, line1, line2, city, postalCode, state, country);
+    Address address1 = new Address(id, name, line1, line2, city, postalCode, state, country);
+    Address address2 = new Address(id, name, line1, line2, city, postalCode, state, country);
```

Comment:
This test could be combined with the hashcode test and modeled the same way to insure that they stay in sync.